widget/embedded/src/components/FilterSelector/FilterSelectorContent.tsx

```diff
@@ -0,0 +1,57 @@
+import type { PropTypes } from './FilterSelector.type';
+
+import { i18n } from '@lingui/core';
+import {
+  Button,
+  Divider,
+  ListItemButton,
+  Radio,
+  RadioRoot,
+  Typography,
+} from '@rango-dev/ui';
+import React from 'react';
+
+import {
+  centeredFlexContainer,
+  FilterContainer,
+  FilterList,
+} from './FilterSelector.style';
+
+export function FilterSelectorContent(props: PropTypes) {
+  const { filterBy: value, onClickItem } = props;
+  return (
+    <FilterContainer>
+      <div className={centeredFlexContainer()}>
+        <Typography size="small" variant="body">
+          {i18n.t('Status')}
+        </Typography>
+        <Button variant="ghost" size="xsmall" onClick={() => onClickItem('')}>
+          {i18n.t('Reset')}
```

Comment:
The font size of `Reset` does not match Figma

Reply:
Fixed